person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    is_of_age? || @parent_permission
```

Comment:
- [ ] you are calling a method named `is_of_age?` however the method you created is called `of_age?` please make sure you're calling the right method.